### ISSUE TITLE
compile error

### ISSUE THREAD

**Issue by bonito-skipjack** [datenlord/Xline#101]
cargo build --release
Compiling console v0.15.2
error: unnecessary parentheses around match arm expression
   --> curp/src/bg_tasks.rs:191:28
    |
191 |                 Ok(req) => (req),
    |                            ^   ^
    |
note: the lint level is defined here
   --> curp/src/lib.rs:41:5
    |
41  |     warnings, // treat all wanings as errors
    |     ^^^^^^^^
    = note: `#[deny(unused_parens)]` implied by `#[deny(warnings)]`
help: remove these parentheses
    |
191 -                 Ok(req) => (req),
191 +                 Ok(req) => req,
    |

   Compiling indicatif v0.17.2
error: could not compile `curp` due to previous error
warning: build failed, waiting for other jobs to finish...

rustc -V
rustc 1.65.0 (897e37553 2022-11-02)
protoc --version
libprotoc 3.21.11
uname -a
Darwin bcd07439fa51 22.1.0 Darwin Kernel Version 22.1.0: Sun Oct  9 20:15:09 PDT 2022; root:xnu-8792.41.9~2/RELEASE_ARM64_T6000 arm64

**Comment by markcty** [comment]
tl;dr: Please set the rust version to 1.61.0: `rustup override set 1.61.0` and compile again.

Your error is a warning generated by the newest version of clippy. We now employ a strict clippy policy and will treat all warnings as errors.
